Shelfie.Identity/Shelfie.Identity.Api/Validators/LoginModelValidator.cs

```diff
@@ -0,0 +1,19 @@
+﻿using FluentValidation;
+using Shelfie.Identity.Api.Models;
+using Shelfie.Identity.Api.Validators.Messages;
+
+namespace Shelfie.Identity.Api.Validators;
+
+public class LoginModelValidator : AbstractValidator<LoginModel>
```

Comment:
I think this validation could be moved to business logic layer and validate command instead of LoginModel. Then all business rules will be kept in one place - in BusinessLogic layer